spec/features/items_spec.rb

```diff
@@ -1,26 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe Item do
-  before(:each) { sign_in_as('Editor') }
+  let!(:item_category) { create(:item_category) }
+  let!(:item_type) { create(:item_type, :item_category => item_category)}
+
+  before(:each) do
+    sign_in_as('Editor')
+
+    allow_any_instance_of(Item).to receive(:item_type).and_return(item_type)
+  end
 
   it "a new item form with appropriate fields" do
-    visit new_item_path
+    visit new_item_type_item_path(item_type)
     fill_in('item_model', :with => 'Model T')
-    expect(find_field('item_status').value).to have_content('Available')
+    expect(find_field('item_status').value).to have_content('Unassigned')
   end
 
   it "an edit form with values filled in" do
-    @item = create(:item)
-    visit edit_item_path(@item)
+    @item = create(:item, :item_type => item_type)
+    visit edit_item_type_item_path(item_type, @item)
     expect(page).to have_field("item_model", :with => @item.model)
   end
 
   it "a new item form with proper departments filled in" do
-    @department1 = Department.create(name: "Manages items", shortname: "Manages items", manage_items: true)
-    Department.create(name: "Doesn't manage items", shortname: "Doesn't", manage_items: false)
-    visit new_item_path
-    expect(page).to have_select("item_department_id", :options => [@department1.name, ''])
+    department_yes = Department.create(name: "Manages items", shortname: "Manages items", manage_items: true)
```

Comment:
because you're only using the name, and you're hardcoding the name here there's no need to bother with creating `Department` objects. Instead just replace `department_yes.name` with the name you've hardcoded here. Same for `department_no.name`

This simplifies the test, saves two db writes, and makes things run faster.